tailwind.config.js

```diff
@@ -26,6 +28,10 @@ const config = {
             ringWidth: {
                 16: '16px',
             },
+            boxShadow: {
+                regisSubmit:
+                    '0px 0px 2.124147891998291px 9.027628898620605px rgba(233, 86, 130, 0.25)',
```

Comment:
I believe this is for a ring-like shadow, right? You can use `ring-[value]` and `ring-pink/30` for the color.